pkg/operator/bootstrap.go

```diff
@@ -312,17 +312,27 @@ func loadBootstrapCloudProviderConfig(infra *configv1.Infrastructure, cloudConfi
 }
 
 func getPlatformManifests(manifests []manifest, platformName string, lbType configv1.PlatformLoadBalancerType) []manifest {
+	var corednsName string
+	var corefileName string
+	switch platformName {
+	case strings.ToLower(string(configv1.GCPPlatformType)):
```

Comment:
Do we need to also handle other platforms like AWS, Azure as well since they don't fall under on-prem category?

Reply:
Yes, we do. Support for AWS and Azure will be added soon in 4.16. API changes are in progress.

Reply:
ah ok. I thought we already support all platforms.